test/jdk/sun/nio/cs/TestCharsetMapping.java

```diff
@@ -569,11 +569,9 @@ private static Set<CharsetInfo> charsets(Path cslist) throws IOException {
 
     public static void main(String args[]) throws Exception {
         Path dir = Paths.get(System.getProperty("test.src", ".") +
-                             "/../../../../src/java.base/share/data/charsetmapping");
+            "/../../../../../make/data/charsetmapping");
         if (!Files.exists(dir)) {
-            // not inside jdk repo, no mappings, exit silently
-            log.println("Nothing done, not in a jdk repo: ");
-            return;
+            throw new Exception("charsetmapping directory cannot be located.");
```

Comment:
Might be helpful to provide directory location in exception message

Reply:
Good point. Fixed.